packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -229,7 +229,7 @@
                             <oui-action-menu-item
                                 data-ng-if="flushCdnState === 'ok' && !hosting.hasCdn && hosting.offer!== 'START_10_M'"
                                 data-text="{{ ::'hosting_dashboard_service_order_cdn' | translate }}"
-                                data-on-click="setAction('cdn/order/hosting-cdn-order', hostingProxy)"></oui-action-menu-item>
+                                data-href="{{:: $ctrl.$state.href('app.hosting.cdn.order') }}"></oui-action-menu-item>
```

Comment:
Done

Reply:
The routing for these sections is particular (ngInclude from the parent page). And adding a resolve like that in `hosting.cdn` means that it will be resolved for the others pages too :(
That's why I've kept this way.

To do that, we should create a routing `hosting.informations` and refactor the controller (inheritance from the parent $scope), but it's not the scope of this feature :)